content/en/tracing/trace_collection/proxy_setup/_index.md

```diff
@@ -298,10 +298,66 @@ The available [environment variables][3] depend on the version of the C++ tracer
 {{% /tab %}}
 {{% tab "Nginx" %}}
 
-Support for Datadog APM is available for Nginx using a combination of plugins and configurations.
-The instructions below use Nginx from the official [Linux repositories][1] and pre-built binaries for the plugins.
+Datadog APM supports Nginx in multiple configurations:
+- Nginx operated as a proxy with tracing provided by the Datadog module.
+- Nginx operated as a proxy with tracing provided by the OpenTracing module.
+- Nginx as an Ingress Controller for Kubernetes.
 
-## Nginx open source
+## Nginx with Datadog module
+Datadog provides an nginx module for distributed tracing. This module is the
+preferred way to add Datadog distributed tracing to nginx.
+
+### Module installation
+There is one version of the Datadog nginx module for each [nginx Docker image
+tag][13]. Install the module by downloading the appropriate file from the
+[latest nginx-datadog GitHub release][14] and extracting it into Nginx's
+modules directory.
+
+For example, if nginx version 1.23.1 is running on a Debian-based system, then
+the appropriate nginx image tag is [1.23.1][15]. The corresponding Alpine-based
+image is tagged [1.23.1-alpine][16].
+
+```bash
+get_latest_release() {
+  curl --silent "https://api.github.com/repos/$1/releases/latest" | jq --raw-output .tag_name
+}
+NGINX_IMAGE_TAG=1.23.1
+RELEASE_TAG=$(get_latest_release DataDog/nginx-datadog)
+tarball="$NGINX_IMAGE_TAG-ngx_http_datadog_module.so.tgz"
+wget "https://github.com/DataDog/nginx-datadog/releases/download/$RELEASE_TAG/$tarball"
+tar -xzf "$tarball" -C /usr/lib/nginx/modules
+rm "$tarball"
+```
+
+### Nginx configuration with Datadog module
+In the topmost section of the nginx configuration, load the Datadog module.
+
+```nginx
+load_module modules/ngx_http_datadog_module.so;
+```
+
+The default configuration connects to a local Datadog Agent and produces traces
+for all Nginx locations. Specify custom configuration in a `datadog` JSON block
+within the `http` section of the nginx configuration.
+
+```nginx
+load_module modules/ngx_http_datadog_module.so;
+
+http {
+  datadog {
+    "service": "usage-internal-nginx",
+    "sample_rate": 0.1
+  }
+}
+```
+
+For information about fields supported by the `datadog` directive and about
+other configuration directives supported by the module, see the [API
+documentation][17].
+
```

Comment:
Should we add the evaluation order between directive and environment variables?

Reply:
That's covered in the documentation of the environment variables, which is linked in the API documentation.